### ISSUE TITLE
NetCDF failing in Windows tests, blocking merging of PRs

### ISSUE THREAD

**Issue by ijpulidos** [choderalab/openmmtools#527]
For months our CI tests have been failing specifically with the windows tests complaining about issues with accessing netcdf files, such as in https://github.com/choderalab/openmmtools/runs/4155555651?check_suite_focus=true with messages like
```
openmmtools.multistate.multistatereporter: DEBUG: Attempt 4/5 to open C:\Users\RUNNER~1\AppData\Local\Temp\tmpno1p10jq\test_storage.nc failed. Retrying in 2 seconds
openmmtools.utils: WARNING: [WinError 32] The process cannot access the file because it is being used by another process: 'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmpno1p10jq\\test_storage.nc'
```
and
```
OSError: [Errno -51] NetCDF: Unknown file format: b'C:\\Users\\RUNNER~1\\AppData\\Local\\Temp\\tmpm5j5mar2\\test_storage.nc'
```
So, I guess we have to ask ourselves if we want to keep supporting Windows and if so, then we want to set up environments that allow us to test this locally. Personally I use Linux as my development OS. 

This is now blocking some of our current workflow to merge PRs, because we would have to manually override the windows tests if we want things to be merged.

**Comment by mikemhenry** [comment]
For now I'm going to disable the windows tests as a requirement so we can get some of these PRs merged in.

**Comment by ijpulidos** [comment]
I think we can close this one. Solved in #535 .
